### ISSUE TITLE
ReferenceError: navigator is not defined when upgrading from 2.14 to 2.15

### ISSUE THREAD

**Issue by StarpTech** [twilio/twilio-video.js#1503]
- [X] I have verified that the issue occurs with the latest twilio-video.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-video.js/blob/master/CHANGELOG.md).
 - [X] I reviewed the [Common Issues](https://github.com/twilio/twilio-video.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [X] I verified that the [Quickstart](https://github.com/twilio/video-quickstart-js) application works in my environment.
 - [X] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.

The issue https://github.com/twilio/twilio-video.js/issues/1500 was closed prematurely.

**Expected behavior:**

Should not introduce breaking changes. The new behavior breaks server-side rendering e.g in Next.js

**Actual behavior:**

```
ReferenceError: navigator is not defined
    at isSupported (/home/starptech/repositories/test/frontpage/node_modules/twilio-video/es5/util/support.js:132:48)
    at Object.<anonymous> (/home/starptech/repositories/test/frontpage/node_modules/twilio-video/es5/index.js:47:37)
```

**Software versions:**

 - [X] Browser(s): Chrome
 - [x] Operating System: Ubuntu
 - [X] twilio-video.js: 2.15
 - [X] Third-party libraries (e.g., Angular, React, etc.): Next.js 11

**Comment by charliesantos** [comment]
Thanks for submitting @StarpTech . This will be addressed in the next release soon.

**Comment by StarpTech** [comment]
@charliesantos a bit of topic. Does exist a repository for twilio-chat? I've found a typo in `v5.0.0` the code that breaks my entire code.

**Comment by charliesantos** [comment]
@StarpTech , are you referring to this twilio-chat module? https://www.npmjs.com/package/twilio-chat
Also, can you please post a snippet of the type you're seeing? I can try reaching out to the right group.

**Comment by jinnkhan88** [comment]
I am having the same issue. Here is the log 

```
ReferenceError: navigator is not defined
    at isSupported (/app/node_modules/twilio-video/es5/util/support.js:132:48)
    at Object.<anonymous> (/app/node_modules/twilio-video/es5/index.js:47:37)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
   
```

 Browser(s): Chrome
 Operating System: Mac OS Big Sur
 twilio-video.js: 2.15 and 2.14 as well. 
 Third-party libraries (e.g., Angular, React, etc.): Next.js 11

**Comment by charliesantos** [comment]
@jinnkhan88 , thanks for the log. Are you also able to reproduce this on 2.14.0?

**Comment by jinnkhan88** [comment]
yes. This is on both versions 2.14.0 and 2.15.0.

**Comment by jinnkhan88** [comment]
I tried using a dynamic import but that didn't work either it says connect is not defined. 

`
const Video = dynamic(() => import('twilio-video').then((module) => module),{
  ssr:false
});`

But I am sure this is a stupid thing to do. 

I did a named import as well. 

`
const Video = dynamic(() => import('twilio-video').then((module) => module.Video),{
  ssr:false
});`

but that was just me trying to get along. But due to this bug our complete web app is in broken state any help in this regard would be helpful.

**Comment by StarpTech** [comment]
> @charliesantos a bit of topic. Does exist a repository for twilio-chat ? I've found a typo in v5.0.0 that breaks my entire application.

Yes

@jinnkhan88 works in 2.14.0. Specify the version with `2.14.0` and not `^2.14.0`.

**Comment by jinnkhan88** [comment]
thanks let me try that.

**Comment by jinnkhan88** [comment]
thank you @StarpTech it worked.

**Comment by charliesantos** [comment]
This has been fixed on 2.15.1.
